### PR TITLE
Merge dev env staging to dev_env branch

### PR DESCRIPTION
**Pull request by sitaram-kalluri**

### CODE REVIEW

File: .github/workflows/at_tools.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: at_tools
+
+on:
+  pull_request:
+    branches:
+      - master
+      - prod_staging
```
Comment (murali-shris) [inline .github/workflows/at_tools.yaml:12]: we can remove staging branches I think

File: at_ve_doctor/pubspec.yaml [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
 environment:
   sdk: '>=2.8.1 <3.0.0'
 
 dependencies:
-  at_server_status: ^0.1.0
+  at_server_status:
+    git:
+      url: https://github.com/atsign-foundation/at_libraries.git
+      path: at_server_status
+      ref: dev_env_staging
```
Comment (murali-shris) [inline at_ve_doctor/pubspec.yaml:15]: ref is dev_env or dev_env_staging ? in automated flow, we cannot change after we push to dev_env

**Comment by murali-shris** [comment]
unit tests should be run, before merging PR to dev_env

File: at_ve_doctor/pubspec.yaml (hunk 2 above)
Comment (kalluriramkumar) [reply at_ve_doctor/pubspec.yaml:15]: Sure Murali, I will remove the staging branches

**Review by murali-shris — APPROVED** [review]

**Pull request merged**